plugins/plugins.go

```diff
@@ -759,6 +761,22 @@ func (m *Manager) ConsoleLogger() logging.Logger {
 	return m.consoleLogger
 }
 
+// ServerInitialized signals a channel indicating that the OPA
+// server has finished initialization.
+func (m *Manager) ServerInitialized() {
+	close(m.serverInitialized)
```

Comment:
nit: Just realized that it would be easy to make this function idempotent. It's only ever being called once from inside of the runtime package but if someone else was instantiating the manager and somehow called this function twice, they would get a panic due to the channel already being closed. We could use a sync.Once primitive here?